wgkex/config/config_test.py

```diff
@@ -0,0 +1,62 @@
+import unittest
+import mock
+import config
+import yaml
+
+_VALID_CFG = "domains:\n- a\n- b\nmqtt:\n  broker_port: 1883\n  broker_url: http://broker\n  keepalive: 5\n  password: pass\n  tls: true\n  username: user\npubkeys_file: some/file\n"
```

Comment:
I think we don't want the `http://` here as we are opening a `mqtt://` connection maybe use this as an URI string?

See https://github.com/mqtt/mqtt.org/wiki/URI-Scheme

Reply:
Makes sense! Thanks for pointing this out!